pageserver/src/import_datadir.rs

```diff
@@ -87,6 +91,75 @@ pub fn import_timeline_from_postgres_datadir<R: Repository>(
     Ok(())
 }
 
+const EMPTY_GLOBAL_RELATIONS: [u32; 12] = [
+    4060, 2966, 4177, 4175, 6100, 2846, 2964, 4183, 4181, 3592, 6000, 4185,
+];
+
+const EMPTY_PER_DATABASE_RELATIONS: [u32; 49] = [
+    826, 3501, 12831, 2328, 2604, 3381, 3256, 2613, 4163, 2620, 4149, 1418, 4165, 2224, 6106, 4151,
+    6102, 3596, 1417, 3429, 4173, 4143, 4167, 6104, 3430, 4157, 2995, 2611, 4153, 4155, 2830, 3350,
+    12841, 2336, 3439, 3118, 2832, 3576, 4169, 3466, 4159, 12836, 6175, 2834, 4171, 12846, 4147,
+    3598, 4145,
+];
+
+// This is a crutch to fix import from backup
+// taken from old pageserver versions. Old
+// pageserver doesn't store empty relations
+// and doesn't add them to backup tar.
+//
+// So, we add them explicitly, using this command.
+pub fn import_empty_rels<R: Repository>(
+    modification: &mut DatadirModification<R>,
+    spcnode: Oid,
+    dbnode: Oid,
+) -> Result<()> {
+    if spcnode == pg_constants::GLOBALTABLESPACE_OID {
+        for relnode in EMPTY_GLOBAL_RELATIONS.iter() {
+            let rel = RelTag {
+                spcnode,
+                dbnode,
+                relnode: *relnode,
+                forknum: pg_constants::MAIN_FORKNUM,
+            };
+
+            debug!("importing global empty relation {}", rel);
+            if let Err(e) = modification.put_rel_creation(rel, 0) {
```

Comment:
what happens if the rel already exists, and is nonempty?

Reply:
`put_rel_creation` will retutn error, we'll catch it and skip this import
anything else?